index.html

```diff
@@ -13,19 +13,20 @@
 </head>
 <body>
 <!---Header section--->
-<section class="header-section">
+<section class="landing-page">  
   <header>
   <nav>
     <a class="logo" href="#">My logo</a>
     <ul id="list-items">
-      <li><a href="https://github.com/citec-47">Portfolio</a></li>
+      <li><a href="#">Portfolio</a></li>
       <li><a href="#">About</a></li>
       <li><a href="#">Contact</a></li>
     </ul>
-   <img src="./images/union-img.png" alt="union icon">
+   <img id="union" src="./images/union-img.png" alt="union icon">
   </nav> 
 </header> 
-  <section class="bio">
+<div class="header-section">
+  <div class="bio">
     <h1 class="heading">
       I’m Diana<br>
 Glad to see you!
```

Comment:
- [ ] Kindly center the contents of the headline section in the desktop view to match the given template. You are expected to stick to design as much as possible.
### Your design
![image](https://user-images.githubusercontent.com/94551484/221291937-59209748-55f1-4f11-b46f-72ea036ece37.png)
### Figma template
![image](https://user-images.githubusercontent.com/94551484/221292048-5bb96f63-7510-49e3-ab30-5c3d7be52327.png)

Reply:
Hello @Kamba56. Thanks for the feedback. I have fixed the spacing now. Can you kindly confirm if it's better?